estate/tests/common.py

```diff
@@ -0,0 +1,16 @@
+from odoo.tests import Form, TransactionCase
+
+
+class EstatePropertyCommon(TransactionCase):
```

Comment:
Use the module name, please. Because this can be used by all unit tests of any model. 

```suggestion
class EstateTransactionCase(TransactionCase):
```